### DISCUSSION TITLE
Unetr & SwinUnetr bottleneck layers weight freezing for transfer learning

### DISCUSSION THREAD

**Discussion by justinzhang528** [Project-MONAI/MONAI#6494] [Q&A]
Dear MONAI Community,

I want to freeze the weights of the bottleneck layers of [Unetr](https://docs.monai.io/en/stable/_modules/monai/networks/nets/unetr.html#UNETR) for transfer learning:
![image](https://github.com/Project-MONAI/MONAI/assets/47877537/89dec8c5-f7a8-4376-ade9-e7378fe1152d)
I want to set "requires_grad = True" for the weights of the bottleneck layers and "requires_grad = False" for the other layers except the bottleneck, but i have no idea which layers are bottleneck layers and which aren't. Could someone help me please?
I also want like to know about [swinUnetr](https://docs.monai.io/en/stable/_modules/monai/networks/nets/swin_unetr.html#SwinUNETR) for freezing bottleneck layers.
Thanks in advance.

**Comment by KumoLiu** [comment]
Hi @tangy5, could you please share some comments on this one, thanks in advance!
